lib/cooked_post_processor.rb

```diff
@@ -412,7 +412,7 @@ def add_lightbox!(img, original_width, original_height, upload, cropped: false)
       end
 
       if small_upload = loading_image(upload)
-        img["data-small-upload"] = small_upload.url
+        img["data-small-upload"] = UrlHelper.cook_url(small_upload.url)
```

Comment:
Nice 👍 Is that the only spot we missed?

Reply:
It's the only spot I did not miss too. :)